tests/functional/supervisor.conf

```diff
@@ -28,7 +28,7 @@ stdout_logfile=/dev/null
 stdout_logfile_maxbytes=0
 stderr_logfile=/dev/null
 stderr_logfile_maxbytes=0
-command=redis-server
+command=redis-server --protected-mode no
```

Comment:
there are some functional tests that use passwords. we use passwords in production. i'd prefer this match more closely.

Reply:
I looked for the functional tests you're referring to, but I wasn't able to find anything that dealt with Redis passwords.

This was necessitated by errors that popped up on this (newer) version of Redis with the Ubuntu upgrade — see [here](https://stackoverflow.com/questions/41223618/sentinel-running-on-protected-mode). The code up to this point has also run with protected mode off, because protected mode didn't exist in the version of Redis that shipped with Ubuntu Xenial. So this is not a regression.